File: tool/src/ru/fomenkov/telemetry/Telemetry.java
package ru.fomenkov.telemetry;

public class Telemetry {

    private Telemetry() {
    }

    public static void log(String format, Object... args) {
        System.out.printf((format) + "%n", args);
    }

    public static void err(String format, Object... args) {
        System.err.printf((format) + "%n", args);
    }
}

File: plugin/src/ru/fomenkov/plugin/util/Telemetry.kt
package ru.fomenkov.plugin.util


object Telemetry {

    var isVerbose = false

    fun verboseLog(message: String) {
        if (isVerbose) println(message)
    }

    fun verboseErr(message: String) {
        if (isVerbose) System.err.println(message)
    }

    fun log(message: String) = println(message)

    fun err(message: String) = System.err.println(message)
}